lib/find.coffee

```diff
@@ -50,6 +50,9 @@ module.exports =
     @subscriptions.add atom.commands.add 'atom-workspace', 'find-and-replace:use-selection-as-find-pattern', =>
       return if @projectFindPanel?.isVisible() or @findPanel?.isVisible()
 
+    @subscriptions.add atom.commands.add 'atom-workspace', 'find-and-replace:use-selection-as-replace-pattern', =>
```

Comment:
The use-selection-as-find-pattern specs still all pass though. Should this code read like this instead?
```
  handleEventsForSelectionAsReplace: ->
    @subscriptions.add atom.commands.add 'atom-workspace',
      'find-and-replace:use-selection-as-replace-pattern': @setSelectionAsReplacePattern
```